admin/class-purger.php

```diff
@@ -263,7 +263,9 @@ private function _purge_by_options( $post_id, $blog_id, $_purge_page, $_purge_ar
 
 			}
 
-			if ( 'post' === $_post_type ) {
+            $post_types = get_post_types(array('public' => true));
+
+			if ( isset($post_types[$_post_type]) || in_array($_post_type, $post_types) ) {
```

Comment:
Only `in_array()` check would be fine. Also, pass `true` as third argument for strict checking. We need to follow WP coding standards.

```php
if ( in_array( $_post_type, $post_types, true ) ) {
```